gtfs/__main__.py

```diff
@@ -1,25 +1,42 @@
 #!/usr/bin/env python
 """Command line interface for fetching GTFS."""
-import argparse
-import getpass
+import typer
+from chalky import chain
 import logging
-import sys
-
 from prettytable import PrettyTable
-
-from gtfs.utils.FeedSource import FeedSource
+from halo import Halo
+import importlib
+from utils.FeedSource import FeedSource
 import feed_sources
-# import all the available feed sources
-# pylint: disable=I0011,wildcard-import
-from feed_sources import *
 
 logging.basicConfig()
 LOG = logging.getLogger()
 LOG.setLevel(logging.INFO)
+app = typer.Typer()
+sources = list(feed_sources.__all__)
+            
+@app.command()
+def list_feeds(xmin:float, ymin:float, xmax:float, ymax:float):
```

Comment:
how's the command invoked currently? is `xmin` etc each a single argument? if so, can we use `bbox` and have them concatenated as string by a comma? e.g. `-5.2,48.20384,-5.482,50.34235` as `minx,miny,maxx,maxy`? 

nit: a space between the arg and its type, i.e. `xmin: float`

Reply:
Right now u have to write something like this to run the function: `python __main__.py list-feeds <args>`. Yes, each param is passed as a single argument. I will try to improve on that. I too think a single argument like `bbox` would be good.

Reply:
btw, the `__main__.py` has a special function: it makes sure you can execute the package with a `-m` switch, i.e. `python -m gtfs` should work (or `python -m gtfs-fetcher`?).